osu.Game/Tests/Visual/SkinnableTestScene.cs

```diff
@@ -33,14 +33,15 @@ public abstract partial class SkinnableTestScene : OsuGridTestScene, IStorageRes
         private Skin metricsSkin;
         private Skin legacySkin;
         private Skin argonSkin;
+        private Skin argonProSkin;
```

Comment:
Are the changes in this file for future usage or something? I don't see how the pro skin will behave any differently as this look to completely depend on the configuration setting (which is the correct way forward, I believe).

Reply:
Oh, uhm I initially thought that it would only apply to the pro skin, but turns out it should also be applied to the normal one as well.
It might be used for future usage though, as there are some other complaints/suggestions about the pro skin in particular, so it might be useful to make comparisons ig.

Reply:
Not so sure. It's giving a lot less room to view the existing elements and I don't really want to check every usage of this test scene to make sure it works well. Let's revert for now.